x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -127,7 +127,7 @@ export const SetupPage: React.FunctionComponent<{
           >
             <FormattedMessage
               id="xpack.ingestManager.setupPage.missingRequirementsCalloutDescription"
-              defaultMessage="In order to use Fleet, you must enable the following Elasticsearch and Kibana security features."
+              defaultMessage="In order to use central management for Elastic Agents, you must enable the following Elasticsearch and Kibana security features."
```

Comment:
```suggestion
              defaultMessage="To use central management for Elastic Agents, enable the following Elasticsearch and Kibana security features."
```

Reply:
You might be able to drop "Elastic Agents" from this sentence, too (not sure because I can't remember what the whole page looks like).

Reply:
@dedemorton the page looks like this but yes I think it's working 
<img width="1110" alt="Screen Shot 2020-10-02 at 9 14 04 AM" src="https://user-images.githubusercontent.com/1336873/94927957-f9f02a00-0490-11eb-8967-3f004ccfd5b1.png">